decred_wallet/view_controller/SendViewController.swift

```diff
@@ -644,25 +715,162 @@ class SendViewController: UIViewController, UITextFieldDelegate,UITextPasteDeleg
                 let amountToSend = Decimal(Double((updatedString)!)!)
```

Comment:
Crashes on this line randomly 
`Thread 1: Fatal error: Unexpectedly found nil while unwrapping an Optional value`